src/components/author/author.stories.mdx

```diff
@@ -40,30 +58,87 @@ const megan = {
 
 # Author
 
-Displays an author, or set of authors, for article bylines.
+Displays an author, or set of authors, for article or presentation bylines.
+
+## Single Author
+
+A single author displays either the `date` or author `job_title`.
+
+### With `date` defined
+
+If the `date` is defined, it will take precedence and render.
 
 <Canvas>
-  <Story name="Default">
+  <Story name="Single author with date" args={{ count: 1 }}>
     {(args) =>
       template({
-        authors: [gerardo, tyler, megan].slice(0, args.count),
+        authors: authors(args.count),
         date: new Date('March 31, 2021'),
       })
     }
   </Story>
 </Canvas>
 
+### Without `date` defined
+
+If the `date` is _not_ defined, the author `job_title` is rendered.
+
+<Canvas>
+  <Story name="Single author without date" args={{ count: 1 }}>
+    {(args) =>
+      template({
+        authors: authors(args.count),
+      })
+    }
+  </Story>
+</Canvas>
+
+## Multiple Authors
+
+Multiple authors can be displayed as well. The authors' `job_title` values are not rendered when multiple authors are provided to this component.
+
+### With `date` defined
+
+If the `date` is defined, it will render.
+
+<Canvas>
+  <Story name="Multiple authors with date" args={{ count: 2 }}>
+    {(args) =>
+      template({
+        authors: authors(args.count),
+        date: new Date('March 31, 2021'),
+      })
+    }
+  </Story>
+</Canvas>
+
+### Without `date` defined
+
+If the `date` is _not_ defined, no additional information is rendered.
```

Comment:
Is there a reason this is specific to multiple authors?

Maybe we should reformat these stories to start with the simplest use case (no date or title), then build up from there by adding a date, and multiple authors, etc., mentioning any limitations along the way?

Reply:
> Is there a reason this is specific to multiple authors?

Great question, @tylersticka!

Perhaps I've misunderstood the requirements?

Before, the Author component would _always_ render a publish date. 

The Speaker requirements were to [include a job title for single speakers and drop the title for multiple speakers](https://github.com/cloudfour/cloudfour.com-patterns/issues/1463#issuecomment-934891778).

Merging those two, I landed at:
  - If 1 author and `date` is defined: Render author avatar, linked name, and date
  - If 1 author and no `date` is defined: Render author avatar, linked name, and `job_title`
  - If multiple authors and `date` is defined: Render all author avatars, linked names, and date
  - If multiple authors and no `date` is defined: Render all author avatars and linked names

> Maybe we should reformat these stories to start with the simplest use case (no date or title)

I didn't know this was a use case! 😅 If it is, the template needs to be updated to handle this use case. Currently, an empty `<p>` tag is rendered. 🔍 👀 

I can make this change, though! 🏗️

Reply:
It's important to remember that the patterns and components in a design system are meant to _enable_ feature requirements at a minimum, but ideally they should be flexible enough to anticipate _future_ requirements.

So our Author component that we're starting from was designed for blog author bylines, which had a specific set of requirements:

1. They must display 1 or more avatars.
2. They must display 1 or more names.
3. They must include a publication date.

But that doesn't quite gel with the speaker requirements you mentioned:

> include a job title for single speakers and drop the title for multiple speakers

So now, we have three options for how to proceed:

1. We can design a whole new component, which was what the first draft of this PR contained. But that seems a bit unnecessary: There's way more that's _similar_ about speaker listings and author listings than there is different.
2. We can _append_ these requirements to the existing component, which is kind of where we are now. That will keep things pretty DRY and will absolutely meet our immediate needs, but it sort of repeats the mistakes that got us here in the first place: We're thinking so specifically about our _current_ needs, that we aren't thinking about how to meet our _future_ needs.
3. We re-think the original requirements to see if we can satisfy both use cases.

I think option 3 is the way to go. And if we think about it from that perspective, here's how I'd revise those original Author requirements:

1. They must display 1 or more avatars.
2. They must display 1 or more names.
3. They ~must~ **may optionally** include a ~publication~ date **or additional text content to display on a second line**.

Not only is that a simpler set of requirements that will be easier for designers and developers to understand, it broadens the potential application of this pattern to more than blog posts and single events.

Let me know if we need to discuss more with voices!

Reply:
Thanks, @tylersticka, that makes sense! 

What happens if both a date and a job title exist at the same time? Or are you suggesting the job title or date is passed into a generic "second line" property?

The current template props are:
- `authors` array where the `author` object contains the job title
- `date`

Are you suggesting to remove the job title from the author object and instead have:
- `authors` array that only includes avatar, name, link
- `second_line_content` (but a better name) that handles a date, or a job title, or any other string?

Thank you! 😄

Reply:
I'm suggesting that the `date` would be kind of a shorthand property for the `second_line_content` (which I might call `extra` or `meta` or something along those lines).

So if I were to think of this as a Twig template, it might be like this toward the be:

```twig
{% set _extra_content %}
  {% block extra %}
    {% if extra %}
      {{extra}}
    {% elseif date %}
      (Do date stuff here!)
    {% endif %}
  {% endblock %}
{% endset %}
```

Then you can check if `_extra_content` includes anything before outputting the `div`...

```twig
{% if _extra_content|trim %}
  <div>
    {{_extra_content}}
  </div>
{% endif %}
```

Check out the Badge component template if you'd like a similar example.

Reply:
Thanks, @tylersticka!

> Check out the Badge component template if you'd like a similar example.

Ha! I was reviewing the Alert component before I read your suggestion and I forgot about this neat pattern (which is also implemented in Badge).

```twig
{% block content %}
  <p>
    {{message|default('Hello world!')}}
  </p>
{% endblock %}
```
Definitely more flexible. 😉

Reply:
Refactored! https://github.com/cloudfour/cloudfour.com-patterns/pull/1607/commits/e6a86975f1c0a4f1a90bff1443fc84bba849b5c2